code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -506,10 +506,11 @@
 			if(spider.is_busy)
 				eggs = locate() in get_turf(spider)
 				if(!eggs || !isturf(spider.loc))
-					var/egg_choice = enriched ? /obj/structure/spider/eggcluster/enriched : /obj/structure/spider/eggcluster
-					var/obj/structure/spider/eggcluster/new_eggs = new egg_choice(get_turf(spider))
+					var/egg_choice = enriched ? /obj/effect/mob_spawn/spider/enriched : /obj/effect/mob_spawn/spider
+					var/obj/effect/mob_spawn/spider/new_eggs = new egg_choice(get_turf(spider))
 					new_eggs.directive = spider.directive
-					new_eggs.faction = spider.faction.Copy()
+					if(length(spider.faction))
+						new_eggs.faction = string_list(spider.faction)
```

Comment:
Is one of these not a string list?
If they're both you can just do `new_eggs.faction = spider.faction`

Reply:
spider.faction is the separate faction list on the mob itself, which I havent touched in this PR.

Reply:
Done.